client-side/src/components/statistics/graphBar.jsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useQuery } from '@apollo/client';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { useAppSelector } from '../../redux/store.jsx';
+import Loader from '../../stories/loader/loader.jsx'
+import { getVisitedWebsitesByDate, formatDate } from './graphsUtils.js';
+import { GET_USERS } from './queries.js';
+
+export default function GraphBar({ startDate, endDate }) {
+  const users = useQuery(GET_USERS);
+  const user = useAppSelector((state) => state.user.currentUser);

```

Comment:
why you can't take the current user from redux?

Reply:
I took from redux, it isn't right?

Reply:
so why do you need to query all the users?

Reply:
So do I need to change something?

Reply:
I hope you will success to take the user from the redux so this useQuery for getting the users will be unnecessary you will be able to remove it

Reply:
I can remove it and use it from Redux, but I don't understand why the apollo-server that I worked on for so long is needed, it is also possible in graphs not to use it but from Redux